app/assets/stylesheets/custom.css.scss

```diff
@@ -48,23 +56,59 @@ a:hover, a:active {
     color: $dark-clam-tan;
 }
 
+.btn {
+    padding: 5px;
+    padding-right: 10px;
```

Comment:
Line should be indented 2 spaces, but was indented 4 spaces<br>Properties should be ordered padding, padding-left, padding-right